etc/version.lock.json

```diff
@@ -660,7 +660,7 @@
     "version": 2
   },
   "4ed678a9-3a4f-41fb-9fea-f85a6e0a0dff": {
-    "rule_name": "Windows Suspicious Script Object Execution",
+    "rule_name": "Suspicious Script Object Execution",
```

Comment:
You can leave this unchanged. When we lock versions, all rule name changes will occur at the same time

Reply:
the PR fail tests whiteout changing it also in the version.lock.json, is that Ok ?

Reply:
it looks like correcting the invalid eql check fixed the failing test, unless you are getting a different local test failure?

Reply:
restored the name in version.lock.json